cobigen/cobigen-core/src/test/java/com/devonfw/cobigen/unittest/util/ConfigurationUtilTest.java

```diff
@@ -60,7 +61,7 @@ public void testFindTemplatesLocation() throws Exception {
 
       String templatesLocation = templates.getAbsolutePath().replace("\\", "\\\\");
       FileUtils.writeStringToFile(configFile,
-          ConfigurationConstants.CONFIG_PROPERTY_TEMPLATES_PATH + "=" + templatesLocation);
+          ConfigurationConstants.CONFIG_PROPERTY_TEMPLATES_PATH + "=" + templatesLocation, Charset.defaultCharset());
```

Comment:
not a good way, this is OS specific - take UTF-8 charset, if you want to have stable tests